### PR TITLE
finalizing learning_goals README

### PR DESCRIPTION
**Pull request by Mo-Altayeb**
added a description to the file and wrote the collective learning goals. added and formatted the individual learning goals that were sent on slack.

### CODE REVIEW

File: collaboration/learning_goals.md [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
-- Learn python
+- Fully understand python basics and get more familiar with it by following guides
+and asking for help.
+- Practice on documenting and testing on different challenges and solutions.
+- Apply what we learned in the foundations track and get used to the workflows
+we were introduced to.
+- Get more familiar with Git commands, and gain experience interacting with the
+Command Line Interface to finish tasks rather than solely relying on the GUI.
+- Get used to GitHub's UI and how to interact with it and learn how
+to maximize it's capabilities.
```
Comment (RandomProjects-db) [inline collaboration/learning_goals.md:36]: the file does not end with a newline character. To fix this issue, you can add a newline character at the end of the file.

**Review by RandomProjects-db — CHANGES_REQUESTED** [review]
in Markdown refers to the fact that Markdown files should end with a single newline character. In the provided code snippet, the last paragraph in the file does not end with a newline character. To fix this issue, you can add a newline character at the end of the file.

**Review by RandomProjects-db — APPROVED** [review]

File: collaboration/learning_goals.md [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+writing unit tests. And enhance collaboration
+by learning to communicate technical ideas clearly and respectfully.
 
 ## Individual
 
-- Learn to write more efficient and readable code.  
+- Learn to write more efficient and readable code that can be understood
+and used by any programmer regardless of it's complexity.
 - Gain confidence in contributing to team-based coding projects.
-- Learn python
```
Comment (Razan-O-Elobeid) [inline collaboration/learning_goals.md:11]: learn python programming is mentioned in the collactive goals

**Review by Razan-O-Elobeid — CHANGES_REQUESTED** [review]
I noticed there are a few overlaps between the collective and individual goals. It might also be helpful to add names to the individual goals to show each person’s focus and experience level. Putting the collective goals in bullet points could make them easier to read too!

**Pull request merged**